### ISSUE TITLE
PDF review for accessibility from login.gov team

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#58128]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [x] Something else 
 
 *Give a brief description of what your team needs support with.*

Sofia Kirkman from the login.gov team asked for assistance with a PDF review for accessibility. I have introduced Sofia and SK via slack.

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] Yes
  - [x] No

## Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

## When does this work need to be done?
- Estimated launch date:
- Estimated staging review date:
- Content, Accessibility, and IA work needed by:  asap?

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Comment by artsymartha68** [comment]
Tagging @skirkman16 on this ticket

**Comment by SarahKay8** [comment]
Hi @skirkman16 this is the PDF Feedback I have so far.  I ran the accessibility checker using Acrobat DC Pro and there were quite a few failures. 

###  Image Alt Text  
All of these images need `alt text` summarizing what is on the photo. I.E "Screenshot of VA.gov Sign in Page" on the first figure you have. Alt attributes enable screen readers to read the information about on-page images for the benefit of a person with complete lack of sight, visually impaired, or who is otherwise unable to view the images on the page. ![DBAAA588-76EE-4CE8-A889-0FFDB5C3FA15.jpeg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/d7152261-e441-4540-891a-5fa3ea350423). 

### Failed Tab Order 
 ![63AA7C58-8922-4DD0-B54A-0E5747F0A158_4_5005_c.jpeg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/acd2fefb-9163-48ff-b9b8-2e9914e9f49d)

How to fix Tab Order Failed

Go to the Pages panel (Icon on left side)
 Right click on a page thumbnail and select Page Properties
 In the Page Properties dialog box, select the Tab Order tab and then choose Use Document Structure
Click 'OK'
Return to Accessibility check 
right click tab order failed
click "check again"
Document should pass

### Heading Hierarchy - ![E1D4B2E1-A4AC-4C5D-9504-44202E850603_4_5005_c.jpeg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/be95c24d-307b-4543-beed-7896629e9fc8)
I would make all blue headers `H'2s` and make each subordinate header underneath an `H'3` with paragraph tags to follow .

### Artifact Footer tags - ![7AF02EEA-ACB9-4ABE-93E7-A64E8B4AF6F6_4_5005_c.jpeg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/3e5f5066-5ce3-42d4-a449-a1e1ead1ebd7)

Because the footers aren't unique, footers should be made artifacts. Generally speaking, the first occurrence of identical  footers should be tagged but the others not, as they interfere with reading and distract AT users from main content. (Screen reader will read each footer after each page, not necessary) 

### Page Title - ![8134818B-7348-4D03-91A3-B287A03BF814_4_5005_c.jpeg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/7b7a13ed-39b5-450a-aa96-ee7f7c6cd1ff)

PDF needs page Title in Metadata. 
Select File > Properties.
Select the Description tab to view the metadata in the document, including the document information dictionary.
Modify the Title field to add or change the document's Title entry. I.E "Login.gov Account Creation Guide" 

### PDF Language - From File Menu, select Properties.
In the “Document Properties” dialog, click the Advanced tab.
Near the bottom of the dialog, select the document language using the Language combo box. (Select English) 

cc- @artsymartha68 @coforma-terry

**Comment by skirkman16** [comment]
Hi! yes you can close it
